<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="GLWidget" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QSlider" name="xSlider">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="pageStep">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="ySlider">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="pageStep">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="zSlider">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="pageStep">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>zSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
